README.md

```diff
@@ -11,5 +11,4 @@ Claire Oct-8-2020 line 2
 Claire Oct-15-2020 line 3
 Ginn's change: Hopefully you can master Git soon.
 
-
-This is from demo on Mar 17th
\ No newline at end of file
+Created new brach for next demo-Alka
```

Comment:
please add more explanation for the demo

Reply:
updated readmefile as per review